src/components/slider/resources.ts

```diff
@@ -0,0 +1,6 @@
+export const CSS = {
+  labelMinValue: "span.handle__label--minValue",
```

Comment:
how come these have `span` inside them? It doesn't seem like we should need to have the element type in the class